salt/gobblin/templates/mr.pull.tpl

```diff
@@ -50,4 +50,4 @@ metrics.reporting.file.enabled=true
 # want to ingest
 # Don't ingest the avro.internal.testbot topic as it's only an internal PNDA
 # testing topic
-topic.blacklist=__consumer_offsets,avro.internal.testbot
+topic.blacklist=__consumer_offsets
```

Comment:
If avro.internal.testbot topic is removed, you will need to make sure data are cleanup by cleaner process otherwise it will continuously increase HDFS storage usage.